tests/unit/test_charm.py

```diff
@@ -41,6 +41,7 @@ def _database_is_available(self):
                 "uris": "http://dummy",
             },
         )
+        return database_relation_id
```

Comment:
The name of this function doesn't make much sense anymore. I'd also add the return type.